packages/apollo-server-errors/src/index.ts

```diff
@@ -4,8 +4,19 @@ import {
   GraphQLFormattedError,
   Source,
   SourceLocation,
+  printError,
+  formatError,
 } from 'graphql';
 
+declare module 'graphql' {
+  export interface GraphQLErrorExtensions {
+    exception?: {
+      code?: string;
+      stacktrace?: ReadonlyArray<string>;
+    };
+  }
+}
+
 export class ApolloError extends Error implements GraphQLError {
```

Comment:
Might be nice to throw in a comment noting that we'd like to switch to `extends GraphQLError` and look forward to doing so as soon as we drop support for <15.7.

Reply:
done ✅